src/app/services/baw-api/api.interceptor.service.ts

```diff
@@ -46,7 +46,7 @@ export class BawApiInterceptor implements HttpInterceptor {
       return next.handle(request);
     }
 
-    // Don't add these headers to requests to cms service
+    // Don't add these headers to requests to non json requests
     if (request.responseType !== "text") {
```

Comment:
the condition does not match the comment. `aplication/octet-stream` is not text... should it have these headers?

Reply:
I see this is part of the responseType type's definition:

```
  responseType: 'arraybuffer' | 'blob' | 'json' | 'text'
```

Still could clear up conditon/comment